core/app/[locale]/(default)/(auth)/login/forgot-password/_actions/reset-password.ts

```diff
@@ -42,32 +24,29 @@ const ResetPasswordMutation = graphql(`
   }
 `);
 
-interface SubmitResetPasswordForm {
-  formData: FormData;
-  path: string;
-  reCaptchaToken: string;
-}
-
-export const resetPassword = async ({
-  formData,
-  path,
-  reCaptchaToken,
-}: SubmitResetPasswordForm) => {
+export const resetPassword = async (
+  _lastResult: (SubmissionResult & { successMessage?: string }) | null,
```

Comment:
We should avoid merging into the Conform submission result and instead should wrap it like we did for other actions:

```ts
return { lastResult: submission.reply(), successMessage }
```

Reply:
Fixed.